etcd-aws-cluster

```diff
@@ -16,6 +16,12 @@ server_port=${ETCD_SERVER_PORT:-2380}
 add_ok=201
 already_added=409
 delete_ok=204
+delete_gone=410
```

Comment:
Where did this 410 status code come from? When I looked at the docs (https://github.com/coreos/etcd/blob/master/Documentation/v2/members_api.md#delete-a-member) I couldn't find a mention of this.